app/facade/partner_facade.rb

```diff
@@ -0,0 +1,38 @@
+class PartnerFacade
+
+  attr_reader :partner
+
+  def build(partner_params=nil)
+    @partner = Partner.new(permitted_params(partner_params))
+  end
+
+  def load(id)
+    @partner = Partner.find(id)
+  end
+
+  def update(id, partner_params)
+    self.load(id)
+    @partner.attributes = permitted_params(partner_params)
+    @partner
+  end
+
+  def load_all
+    Partner.all
+  end
+
+  def save
+    @partner.save
+  end
```

Comment:
the controller does not use the `save` from facade. This method is unused, we forgot to delete it.
Save is handled by the controller as it was before.